app.js

```diff
@@ -9,6 +9,8 @@ const bodyParser = require('body-parser')
 
 // Mount db connection
 require('./config/db').connect()
+  .then(() => console.log('db connection open'))
+  .catch(() => process.exit(1))
```

Comment:
Yes, agreed. 
There is right now also another service setup, the SheetsService, that requires successful connection (google spreadsheets) before starting up. 
This is a good catch, and a debt to fix, but probably not for this PR (since this belongs to the main app and the PR only for the CLI tool..., and it doesn't hurt very much at the moment). But a refactor will be needed to properly setup all the services and catch/handle possible errors to exit early.